endpoints.json

```diff
@@ -55,21 +59,39 @@
           "author": "icellusedkars",
           "votes": 0,
           "article_id": 3,
-          "created_at": "2020-09-20 12:09",
+          "created_at": "2020-09-20 12:09:00",
           "body": "Ambidextrous marsupial"
         },
         {
           "comment_id": 10,
           "author": "icellusedkars",
           "votes": 0,
           "article_id": 3,
-          "created_at": "2020-06-20 08:06",
+          "created_at": "2020-06-20 08:06:00",
           "body": "git push origin master"
         }
       ]
     }
-  }
+  },
 
+  "POST /api/articles/:article_id/comments": {
+    "description": "Creates a new comment by the username provided, attatched to the article_id provided",
+    "request body required": {
```

Comment:
I would camel case this to keep this consistent with the other keys and make it easier for the end user to access this key, it can just be requestBody